package.json

```diff
@@ -52,7 +52,7 @@
     "graphql-tag": "^2.12.6",
     "jsonwebtoken": "^8.5.1",
     "jwks-rsa": "^2.1.4",
-    "next": "12.2.0",
+    "next": "12.2.3",
```

Comment:
Why next version is bumped in this PR?

Reply:
I had some issues with hydratation that seemed to be a bug in next, that's why I bumped it.

Reply:
Ok, for the future, try to avoid mixing features with updating dependencies. If we have a regression it will hard to find what was the actual reason (hence small commits, often, work best)